pkg/workflows/create.go

```diff
@@ -366,10 +366,6 @@ func (s *InstallEksaComponentsTask) Run(ctx context.Context, commandContext *tas
 	datacenterConfig := commandContext.Provider.DatacenterConfig(commandContext.ClusterSpec)
 	machineConfigs := commandContext.Provider.MachineConfigs(commandContext.ClusterSpec)
 
-	// this disables create-webhook validation during create
-	commandContext.ClusterSpec.Cluster.PauseReconcile()
```

Comment:
We did not support this for upgrade before? I expected us to have had this already.

Reply:
we do not have this upgrade validation in cluster webhook before
https://github.com/aws/eks-anywhere/blob/main/pkg/api/v1alpha1/cluster_webhook.go#L87

Reply:
That was because we only supported a subset of changes right? For some reason I thought we always paused the controller for upgrade too because we don't want the controller to mess the changes that the CLI is making.

Reply:
> we always paused the controller for upgrade too because we don't want the controller to mess the changes that the CLI is making

we already do that. 

this logic is to add paused annotation in the cluster spec that will be applied. The existing object in cluster already has paused annotation, but we need to keep that annotation in order to pass the validation.